client/data/authorizations/actions.ts

```diff
@@ -39,3 +71,58 @@ export function updateAuthorizationsSummary(
 		query,
 	};
 }
+
+export function* submitCaptureAuthorization(
+	paymentIntentId: string,
+	orderId: number
+): Generator< unknown | Authorization > {
+	try {
+		yield dispatch( STORE_NAME, 'startResolution', 'getAuthorization', [
+			paymentIntentId,
+		] );
+
+		const result = yield apiFetch( {
+			path: `/wc/v3/payments/orders/${ orderId }/capture_authorization`,
+			method: 'post',
+			data: {
+				payment_intent_id: paymentIntentId,
+			},
+		} );
+
+		const authorization = {
+			payment_intent_id: ( result as any ).id,
```

Comment:
can we use a type other than `any` here? Or I think `apiFetch`'s return value isn't typed?

Reply:
Yes, I struggled with this one because the return type of apiFetch is `unknown`. So I ended using `any`. I went for a different approach in this commit [4c42d1e](https://github.com/Automattic/woocommerce-payments/pull/4961/commits/4c42d1efc8b2b2300b8c3d0e83c0ed7cdeec6ced).